truco-back/src/app.ts

```diff
@@ -3,10 +3,7 @@ import path from "path";
 
 const app = express();
 
-app.use(express.static(path.join(__dirname, 'public')));
-
-app.use("/", (req, res) => {
-    res.sendFile(path.join(__dirname, '/public/index.html'));
+app.get("/api", (req, res) => {
+    res.send("Hello World");
 })
-
```

Comment:
Without this the app is no longer serving the static files from the backend which means the app will be separated into two different servers, one for the frontend and one for the backend. That's fine during development as we will use CORS, but we should not delete it because in the PROD version of the app there will only be one server, and without this lines it won't be possible

Reply:
just to make it clear, the api WILL be at xxxxxx/api but when a user enters xxxxxx/ they should see the frontend homepage not a `Cannot GET /` message

![image](https://github.com/maxogod/Truco/assets/105026197/1f660c65-5f76-4e81-9efb-229ba2e0aee4)
![image](https://github.com/maxogod/Truco/assets/105026197/4c08cb68-c560-4245-8aad-20636517b1d4)

Reply:
With this implementation, nginx is used as an inverse proxy. If you go to localhost:3050/api you will find the backend
if you go to localhost:3050, you will find the frontend.